static-assets/components/cstudio-forms/data-sources/video-S3-upload.js

```diff
@@ -57,8 +57,8 @@ YAHOO.extend(CStudioForms.Datasources.VideoS3Upload, CStudioForms.CStudioFormDat
 
     var callback = {
       success: function (fileData) {
-        var uri = fileData;
-        var fileExtension = fileData.split('.').pop();
+        var uri = fileData.url ? fileData.url : fileData;
+        var fileExtension = uri.split('.').pop();
```

Comment:
What's the reason for the property being one thing or the other? If it is the API, it documented there?

Reply:
According to the API, it's the `fileData.url`, though I got confused at it is that way in the image datasource. Should I update them as part of this PR?